### PR TITLE
Screen shot

### LINKED ISSUE #4: Game Preview
A screenshot of game will help

### PR DESCRIPTION
**Pull request by ghost**
added screenshot. closes #4.

### CODE REVIEW

**Comment by ghost** [comment]
@narendravardi bro, look into this.

**Comment by narendravardi** [comment]
@vamsidharBada, Thanks for the PR.

Just a few modifications, 
- **f543922** commit is already resolved in #2. Please analyze why this commit got added to this PR.
- What do you think about embedding screenshot in READ_ME instead of a separate folder? The repo page would look very welcoming :)
- https://github.com/vamsidharBada/2048-Game/tree/screen_shot, this is the link to the repo page of forked 2048 game, in **screen_shot** branch. Is this how you expected the READ_ME file to be?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
 # 2048-Game
 This is a 2048 game developed using Swings in java.
+<<<<<<< HEAD
```
Comment (narendravardi) [inline README.md]: This is a merge commit.

Please take care of this.

**Review by narendravardi — CHANGES_REQUESTED** [review]

**Review by narendravardi — APPROVED** [review]

**Comment by narendravardi** [comment]
@vamsidharBada also please embed this screen shot in the read me file.

Thanks for the PR!!

**Pull request merged**